optuna/samplers/_nsga2/crossover.py

```diff
@@ -74,62 +80,18 @@ def try_crossover(
             for parent in parents
         ]
     )  # Parent individual with NUMERICAL_DISTRIBUTIONS parameter.
-    if crossover_name == "uniform":
-        child_params_array = _uniform(
-            parents_numerical_params_array[0],
-            parents_numerical_params_array[1],
-            rng,
-            swapping_prob,
-        )
-    elif crossover_name == "blxalpha":
-        alpha = 0.5
-        child_params_array = _blxalpha(
-            parents_numerical_params_array[0], parents_numerical_params_array[1], rng, alpha
-        )
-    elif crossover_name == "sbx":
-        if len(study.directions) == 1:
-            eta = 2
-        else:
-            eta = 20
-        child_params_array = _sbx(
-            parents_numerical_params_array[0],
-            parents_numerical_params_array[1],
-            rng,
-            numerical_distributions,
-            eta,
-        )
-    elif crossover_name == "vsbx":
-        if len(study.directions) == 1:
-            eta = 2
-        else:
-            eta = 20
-        child_params_array = _vsbx(
-            parents_numerical_params_array[0], parents_numerical_params_array[1], rng, eta
-        )
-    elif crossover_name == "undx":
-        sigma_xi = 0.5
-        sigma_eta = 0.35 / np.sqrt(len(parents_numerical_params_array[0]))
-        child_params_array = _undx(
-            parents_numerical_params_array[0],
-            parents_numerical_params_array[1],
-            parents_numerical_params_array[2],
-            rng,
-            sigma_xi,
-            sigma_eta,
-        )
-    elif crossover_name == "spx":
-        epsilon = np.sqrt(len(parents_numerical_params_array[0]) + 2)
-        child_params_array = _spx(parents_numerical_params_array, rng, epsilon)
-    else:
-        assert False
 
-    child_numerical_params = numerical_transform.untransform(child_params_array)
+    child_numerical_array = crossover.crossover(
+        parents_numerical_params, rng, study, numerical_search_space
+    )
+    child_numerical_params = numerical_transform.untransform(child_numerical_array)
     child_params.update(child_numerical_params)
+
     return child_params
 
 
 def crossover(
```

Comment:
How about change the name of this function or its argument `crossover`?

Reply:
Went with `perform_crossover` for the function name in bdf983175e6cb58c3c4be0eeae5221af7eb23309.